Sources/KeyboardSupport/KeyboardAccessory.swift

```diff
@@ -29,5 +29,9 @@ public typealias KeyboardInputAccessory = KeyboardAccessory
 
 /// Represents something that has keyboard accessory navigation options.
 public protocol KeyboardAccessory: class {
+    var nextButton: UIBarButtonItem? { get set }
+    var backButton: UIBarButtonItem? { get set }
+    var doneButton: UIBarButtonItem? { get set }
```

Comment:
I dropped these in here as opposed to using a separate protocol or subclass because the KeyboardAccessoryDelegate contains "next, back, and done" pressed. It made me think that those were more inherent to the toolbar. 

Storing the buttons allows them to be enabled, disabled, or removed pending on the navigation situation by the autonav.

Reply:
I could create a NavigatingKeyboardAccessory that would add the next and back buttons while leaving the done button in this one. 

OR
Just stick all three into a NavigatingKeyboardAccessory

OR 
Stick all three into an AutoNavigatingKeyboardAccessory protocol to give it more indication that it should be used with the AutoNav. 

Thoughts?

Reply:
👍 
I've left KeyboardToolbar as implementing both protocols, and changed Nav and AutoNav to accept the protocol in the form of a new typealias instead of the concrete class. 

```public protocol NavigatingKeyboardAccessory: KeyboardAccessory {
    var nextButton: UIBarButtonItem? { get set }
    var backButton: UIBarButtonItem? { get set }
    
    func setNextAndBackButtonsHidden(_ hidden: Bool)
}

public typealias KeyboardAccessoryView = KeyboardAccessory & UIView
public typealias NavigatingKeyboardAccessoryView = NavigatingKeyboardAccessory & UIView```

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/179be6970fc071bb9ce43417b249bc48c2795ee8